conan/tools/scm/git.py

```diff
@@ -24,6 +24,7 @@ def run(self, cmd):
 
         :return: The console output of the command.
         """
+        self._conanfile.output.verbose(f"Running git {cmd}")
```

Comment:
The ``Conanfile.run()`` already has some output, with a specific color, etc, why not the same format and color?

TODO: Is there any risk that the `cmd` will contain the URL that includes credentials?